pnpm-lock.yaml

```diff
@@ -2,8 +2,8 @@ lockfileVersion: 5.4
 
 specifiers:
   '@bazel/bazelisk': ~1.12.1
-  '@types/debug': ^4.1.7
-  '@types/node': ^18.8.5
+  '@types/debug': ^4.1.10
+  '@types/node': ^18.15.13
```

Comment:
@mrbbot ... any reason we can't bump up to Node.js v20?

Reply:
Added a workaround I had in mind to make the latest Node version available... Was able to upgrade to 20.8, so almost up to date!